### DISCUSSION TITLE
Upload of Physiology parameter for a Cat and Cattle model

### DISCUSSION THREAD

**Discussion by KirstenBoerngen** [Open-Systems-Pharmacology/Forum#953] [Models]
Attached a list with several parameters that could be used to implement a Cat and Cattle model in PK Sim. However, references for the individual parameter are still missing. I will try to add these information within the next weeks.

[Cattle_Cat_Parameters_upload.xlsx](https://github.com/Open-Systems-Pharmacology/Forum/files/7005828/Cattle_Cat_Parameters_upload.xlsx)

**Comment by Incei** [comment]
Dear @KirstenBoerngen  @Yuri05  Is there any documentation reported in the meantime on the models? 
Cheers,

**Reply by StephanSchaller** [reply]
Hi Ibra. No, not yet. Are you utilizing the models?

**Reply by Incei** [reply]
Hi Stephan, there is interest by others. ;) Any plans to publish in the next weeks/months? 
We could take it off-line if you know more. 
Cheers

**Comment by CorentinMit** [comment]
Hi all,
On a related subject, I intend to use the cattle model for some IV simulations. I was wondering what the meaning of 'experimental' is. Is it because it has not been validated on a proper dataset yet?
Thank you very much
Corentin

**Reply by StephanSchaller** [reply]
> because it has not been validated on a proper dataset yet?

Close, but not entirely, it has been validated on "internal" data, when it was developed. But it has not been evaluated on a public dataset such that this evaluation could be scrutinized by the public.
